fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_FCU.xml

```diff
@@ -291,14 +291,10 @@
                     }
                     (&gt;H:A320_Neo_CDU_VS)
                 </ANTICLOCKWISE_CODE>
-                <PULL_CODE>
+                <LEFT_SINGLE_CODE>
                     (&gt;H:A320_Neo_FCU_VS_PULL)
                     (&gt;H:A320_Neo_CDU_VS)
-                </PULL_CODE>
```

Comment:
Shouldn't we also remove then the code that handled this? E.g. https://github.com/flybywiresim/aircraft/blob/master/fbw-a380x/src/systems/instruments/src/FCU/Managers/VerticalSpeedManager.ts#L63

Reply:
Fair point. We might be able to reused some stuff for the ALT button but probably not that.